### ISSUE TITLE
Accessibility contrast issues

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#196]
I am plugging away on accessibility and am now running into some contrast errors. I have added some custom CSS to fix the majority of the issues, however there are issues in the site header that I believe will take your expertise.

If you go to the [WAVE tool for our page](http://wave.webaim.org/report#/www.jccmi.edu) and click on the **contrast** tab at the top, a lot of our contrast issues are coming from the site header. If you click on the red low contrast error buttons it will jump to the links and show the colors it's reading. 

The tool is reading that the background of the header is #fff. When I went inspected this in google developer tools I was able to toggle off the #fff background color and the links changed colors to dark - so the WAVE tool is reading that the background #fff and the link color #fff are one in the same and sending an error. I'm not sure how this header is built and if it's possible to code it so the accessibility tool can read that the background color is actually dark and the link colors are white so it doesn't throw out an error?

I'm not sure how the government is going to check the accessibility of the site, whether they will run a tool and then go in an see for themselves, but I would love to eliminate as many errors as possible!

Thanks!

**Comment by joelworsham** [comment]
The technical reason for this is that I was using a pseudo-element of `::before` to add the background to the header. So, the tool did not see this and followed up the tree to the nearest element with a background color, which is the body, which is #fff.

I honestly cannot recall *why* I opted for using a pseudo-element instead of just giving the header element the background. I'm certain there was a reason, but it's possible that the reason is no longer present. I just changed it so the header itself has the background. If you notice any oddities in the header regarding this, let me know.

**Comment by banksashleym** [comment]
Sounds good, thanks! I just did another scan and it's still showing the contrast errors. Will it take some time before the changes go into effect? I really appreciate all of your help with all of this accessibility stuff!

![capture](https://user-images.githubusercontent.com/15264315/31833668-51be97c0-b599-11e7-8978-4dbe6e73f18a.JPG)

**Comment by joelworsham** [comment]
I haven't yet pushed the changes to the live site. Going to soon!

**Comment by banksashleym** [comment]
lol, I knew I jumped the gun too soon! Thank you!
